polytope_server/common/staging/s3_staging.py

```diff
@@ -87,6 +89,7 @@ def __init__(self, config):
             config=botocore.config.Config(
                 max_pool_connections=50,
                 s3={"addressing_style ": "path"},
+                signature_version="s3v4",
```

Comment:
Perhaps we should check with ECMWF if this is okay for them but this wait until we open the PR on their side